website/src/content/development/adding-typescript.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Adding TypeScript
+---
+
+[TypeScript](https://www.typescriptlang.org/) is a typed superset of JavaScript that compiles to plain JavaScript.
+
+Custom Applications have full support for developing applications in TypeScript.
+
+<Info>
+
+This feature is available from version `>= 21.8.0`.
+
+</Info>
+
+To start a new Custom Application project with TypeScript, you can use the TypeScript starter template:
+
+```console noPromptLines="2-3"
+npx @commercetools-frontend/create-mc-app@latest \
+  my-new-custom-application-project \
+  --template starter-typescript
+```
+
+The TypeScript starter template is the same as the standard JS starter template in terms of functionality but it includes the additional TypeScript setup.
+
+# Configuration
```

Comment:
Here we explain about the needed configuration but, at least for me, it seems like this is something that I would need to do when creating a new Custom Applications.

Perhaps we could add a note at the end of the section explaining all of these steps are already preconfigured in our starter template and users don't need to worry about setting them up.

Reply:
Thanks, updated here: https://github.com/commercetools/merchant-center-application-kit/pull/2624/commits/785d879b8aa392d7235f38076fef2b6d3e5adbee